lib/Sema/TypeCheckStorage.cpp

```diff
@@ -2705,6 +2706,11 @@ static void typeCheckSynthesizedWrapperInitializer(VarDecl *wrappedVar,
 
   initializer = result->getAsExpr();
 
+  // Contextualize the initializer which is a local variable with defaultInit or
+  // gets an independent initializer. The rest of initializer contextualizing
+  // will be done in visitPatternBindingDecl.
```

Comment:
After this return statement, the function also checks actor isolation and initializer effects. Do we have tests to make sure the cases where `contextualize` is `false` still perform actor isolation and effects checking? We might also want to call the parameter something else since it skips more than contextualizing closures.